tox.ini

```diff
@@ -61,7 +61,15 @@ commands =
     sphinx-build {posargs:-E} -b html docs dist/docs
     sphinx-build -b linkcheck docs dist/docs
 
+[testenv:pdf]
```

Comment:
Does it take a long time to build the PDF? If not, maybe just build both HTML and PDF in the same action.